src/mainloop.rkt

```diff
@@ -449,9 +461,9 @@
 
   (timeline-event! 'soundness)
 
-  (define best-annotated
-    (first (add-soundiness (list best) (*pcontext*) (*context*))))
+  (match-define (cons best-annotated rest-annotated)
+    (add-soundiness (cons best rest) (*pcontext*) (*context*)))
```

Comment:
I'm not sure how soundiness works but what does this change do?

Reply:
Gets proofs for _all_ output expressions